docs/src/assets/scss/components/search.scss

```diff
@@ -70,7 +70,7 @@
     width: 100%;
     border-radius: 0 0 var(--border-radius) var(--border-radius);
     border: 1px solid var(--divider-color);
-    position: rekative;
+    position: relative;
```

Comment:
Was going to try running stylelint to see if there is any other odd ones later

Reply:
We have a similar PR open https://github.com/eslint/eslint/pull/16226